app/design/frontend/Magento/elogic-theme/web/css/source/_variables.less

```diff
@@ -30,8 +32,8 @@
 
 //  Responsive breakpoints
 
-@screen__m_s: 480px;
-@screen__m_l: 767px;
+@screen__xl: 1200px;
+@screen__lg: 1440px;
 
```

Comment:
These breakpoints exist already in code/lib/web/css/source/lib/variables/_responsive.less:
```
@screen__l: 1024px;
@screen__xl: 1440px;
```